climbing_gym/demo/sign_request_item.xml

```diff
@@ -0,0 +1,33 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<odoo noupdate="1">
+    <function name="sign" model="sign.request.item" context="{'install_mode': True}">
```

Comment:
we still need to understand the warning (and install_mode does not work here, unlike sale.order confirmation)